Source/CTPanoramaView.swift

```diff
@@ -203,6 +204,9 @@ fileprivate extension CMDeviceMotion {
         sceneView.backgroundColor = UIColor.black
         sceneView.delegate = self
         sceneView.isPlaying = true
+        if #available(iOS 9.0, *) {
+            _ = sceneView.audioEngine
```

Comment:
Its a known Apple bug being exposed by the CTPanoramaView

https://forums.developer.apple.com/thread/65793
http://stackoverflow.com/questions/39568433/what-is-core-audio-error-561015905-and-why-does-it-happen-when-i-use-the-lock-bu

Only reason iOS 8 is still supported is I didn't make changes from the fork to drop support.